src/holofuel/contexts/useFlashMessageContext.js

```diff
@@ -0,0 +1,19 @@
+import React, { createContext, useContext, useState } from 'react'
```

Comment:
this was initially three separate files, but it ended up making sense to put them all in one place. This is a candidate for our general global state pattern, so would love for it to
a) be readable and easy to think about and use
b) be easy to set up new pieces of global state

I anticipate that this file might find a new home, and the exact structure of how we're doing this might change as the pattern matures, but this seems a fine start. 

So much so that after review I think I'll refactor screenWidth to use the same pattern.

Reply:
Makes sense.